test/Microsoft.AspNet.Security.Test/project.json

```diff
@@ -18,7 +20,8 @@
     "frameworks": {
         "aspnet50": {
             "dependencies": {
-                "Shouldly": "1.1.1.1"
+                "Shouldly": "1.1.1.1",
+                "System.Security.Claims": "1.0.0-*"
```

Comment:
This test project doesn't support aspnetcore50 so you don't need to list any dependencies in the framework specific section. If we decide to test core later we'll have to deal with the dependencies then.